clusters/l3-sqnc/capacity/open-api-merger/values.yaml

```diff
@@ -0,0 +1,15 @@
+logLevel: info
+paths:
+  - http://capacity-identity-service-sqnc-identity-service:3000/api-docs
+  - http://capacity-matchmaker-api:3000/api-docs
+baseUrl:
+  - https://capacity.dsch-l3.com
+ingress:
```

Comment:
should the ingress not have a class name? For example see https://github.com/digicatapult/sqnc-flux-infra/blob/0556b4a54661e2748b8bedc8ef0bd6b84c0f3e0b/clusters/l3-sqnc/capacity/matchmaker-api/values.yaml#L15. We should probably also disable this until we understand the auth situation?

Reply:
I think so yes although if nothing else it's for consistency with the other services. Similarly we should disable the ingress here until auth is sorted I think